frontend/.eslintrc.js

```diff
@@ -108,10 +108,9 @@ module.exports = {
 		// eslint rules need to remove
 		'@typescript-eslint/no-shadow': 'off',
 		'import/no-cycle': 'off',
-
 		'prettier/prettier': [
 			'error',
-			{},
+			{ endOfLine: 'auto' },
```

Comment:
@csudani7 please remove this change ... if we are not targeting this issue in current PR

Reply:
@palashgdev Forgot to un stash on mass files of commit. I will remove in next commit.